pkgs/development/ocaml-modules/slug/default.nix

```diff
@@ -0,0 +1,37 @@
+{ lib, fetchFromGitHub, buildDunePackage
+, re, uutf, uunf, uuseg
+, alcotest
+}:
+
+buildDunePackage rec {
+  pname = "slug";
+  version = "1.0.1";
+
+  duneVersion = "3";
+
+  src = fetchFromGitHub {
+    owner = "thangngoc89";
+    repo = "ocaml-slug";
+    rev = version;
+    sha256 = "sha256-pIk/0asSyibXbwmBSBuLwl2SS9aw6dNDDvwO+1VJGf8=";
+  };
+
+  propagatedBuildInputs = [
+    re
+    uutf
```

Comment:
I don't think this is particularly important, but it looks like `uutf` is not needed to be in `propagatedBuildInputs`.

`requires = "re uunf.string uuseg.string"`

I get this by building the package and then running `cat ./result/lib/ocaml/4.14.1/site-lib/slug/META`.

Reply:
You seem to be right! I don't understand; I didn't take it from the `.opam` file but I added it after some tests. What in the world did I do? :thinking: